cmd/main.go

```diff
@@ -269,6 +270,18 @@ func main() {
 	usageManager := usage.NewManager(log, notificationStream)
 	ocmClient := getOCMClient(log)
 
+	var decodedECPublicKeyPEM, decodedECPrivateKeyPEM []byte
+	if Options.Auth.AuthType == auth.TypeAgentLocal {
+		decodedECPublicKeyPEM, err = base64.StdEncoding.DecodeString(Options.Auth.ECPublicKeyPEM)
+		Options.Auth.ECPublicKeyPEM = string(decodedECPublicKeyPEM)
+		failOnError(err, "Error decoding public key:")
+
+		decodedECPrivateKeyPEM, err = base64.StdEncoding.DecodeString(Options.Auth.ECPrivateKeyPEM)
+		Options.Auth.ECPrivateKeyPEM = string(decodedECPrivateKeyPEM)
+		os.Setenv("EC_PRIVATE_KEY_PEM", string(decodedECPrivateKeyPEM))
+		failOnError(err, "Error decoding private key:")
+	}
```

Comment:
Do all this in the agent local auth authenticator.

Also I'd add a comment or something about why you can't use the string directly.
Ideally the env var would be the PEM (as the name implies), but I know you had some issues with multiline env vars in systemd units.